ansible_ai_connect/ai/api/model_pipelines/http/pipelines.py

```diff
@@ -238,74 +239,40 @@ def get_streaming_http_response(
-            conversation_id = params.conversation_id
-            provider = params.provider
-            model_id = params.model_id
-            system_prompt = params.system_prompt
-            media_type = params.media_type
-
-            data = {
-                "query": query,
-                "model": model_id,
-                "provider": provider,
-            }
-            if conversation_id:
-                data["conversation_id"] = str(conversation_id)
-            if system_prompt:
-                data["system_prompt"] = str(system_prompt)
-            if media_type:
-                data["media_type"] = str(media_type)
-
-            async with session.post(
-                self.config.inference_url + "/v1/streaming_query",
-                json=data,
-                headers=headers,
-                raise_for_status=False,
-            ) as response:
-                if response.status == 200:
-                    async for chunk in response.content:
-                        try:
-                            if chunk:
-                                s = chunk.decode("utf-8").strip()
-                                if s and s.startswith("data: "):
-                                    o = json.loads(s[len("data: ") :])
-                                    if o["event"] == "error":
-                                        default_data = {
-                                            "response": "(not provided)",
-                                            "cause": "(not provided)",
-                                        }
-                                        data = o.get("data", default_data)
-                                        logger.error(
-                                            "An error received in chat streaming content:"
-                                            + " response="
-                                            + data.get("response")
-                                            + ", cause="
-                                            + data.get("cause")
-                                        )
-                        except JSONDecodeError:
-                            pass
-                        logger.debug(chunk)
-                        yield chunk
-                else:
-                    logging.error(
-                        "Streaming query API returned status code="
-                        + str(response.status)
-                        + ", reason="
-                        + str(response.reason)
-                    )
-                    error = {
-                        "event": "error",
-                        "data": {
-                            "response": f"Non-200 status code ({response.status}) was received.",
-                            "cause": response.reason,
-                        },
-                    }
-                    yield json.dumps(error).encode("utf-8")
-                    return
+        query = params.query
```

Comment:
JFYI, if/when this becomes a _real_ PR a new `ModelPipeline` implementation and related code should be written. I suspect this is the quickest way to try your prototype code... but is not desirable. You could use `http` as an example/template for a new `llama-stack` provider.